.github/actions/frontend-deploy/action.yml

```diff
@@ -0,0 +1,48 @@
+### !!!
+# This file is automatically generated using Terraform.
+# Do not update it manually. If changes need to be made,
+# please request help from a maintainer to generate the
+# updated version.
+### !!!
+
+name: openverse/staging-frontend-nuxt-deploy
+description: Update the task definition for a staging-frontend-nuxt deployment
+
+inputs:
+  tag:
+    required: true
+    description: Image tag to deploy.
+
+runs:
+  using: "composite"
+
+  steps:
+    - name: Configure AWS credentials
+      uses: aws-actions/configure-aws-credentials@v1
+      with:
+        aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        aws-region: us-east-2
+
+    - name: Download task definition
+      with: shell
+      run: |
+        aws ecs describe-task-definition \
+          --task-definition staging-frontend \
+          --query taskDefinition > task-definition.json
+
+    - name: Fill in the new image ID in the Amazon ECS task definition
+      id: task-def
+      uses: aws-actions/amazon-ecs-render-task-definition@v1
+      with:
+        task-definition: task-definition.json
+        container-name: nuxt
+        image: ghcr.io/wordpress/openverse-frontend:${{ inputs.tag }}
+
+    - name: Deploy Amazon ECS task definition
+      uses: aws-actions/amazon-ecs-deploy-task-definition@v1
+      with:
+        task-definition: ${{ steps.task-def.outputs.task-definition }}
+        service: staging-frontend
+        cluster: staging-default
+        wait-for-service-stability: true
```

Comment:
I don't know much about what stability is defined as, but I want to know that as I recall the default wait time is something like 30 minutes? Edit: found it, yes 30. https://github.com/aws-actions/amazon-ecs-deploy-task-definition/blob/master/action.yml